docker-compose.yml

```diff
@@ -7,10 +7,14 @@ services:
     container_name: zkevm-sequencer
     image: zkevm-node
     environment:
-      - ZKEVM_NODE_DATABASE_USER=test_user
-      - ZKEVM_NODE_DATABASE_PASSWORD=test_password
-      - ZKEVM_NODE_DATABASE_NAME=test_db
-      - ZKEVM_NODE_DATABASE_HOST=zkevm-db
+      - ZKEVM_NODE_STATEDB_USER=state_user
```

Comment:
Suggestion: this config is already defined in config files. We can rely on it instead of having it on the docker-compose